lib/ood_core/job/adapters/pbspro.rb

```diff
@@ -9,18 +9,18 @@ class Factory
 
       # Build the PBS Pro adapter from a configuration
       # @param config [#to_h] the configuration for job adapter
-      # @option config [Object] :host (nil) The batch server host
+      # @option config [Object] :submit_host (nil) The batch server host
       # @option config [Object] :exec (nil) Path to PBS Pro executables
       # @option config [Object] :qstat_factor (nil) Deciding factor on how to
       #   call qstat for a user
       # @option config [#to_h] :bin_overrides ({}) Optional overrides to PBS Pro client executables
       def self.build_pbspro(config)
         c = config.to_h.compact.symbolize_keys
-        host = c.fetch(:host, nil)
+        submit_host = c.fetch(:submit_host, "")
```

Comment:
We want to keep both host and submit host as they are different values.

- `host` is the host of the batch server we are submitting the job to
- `submit_host` is the host where we are executing the job submission command